### ISSUE TITLE
Focus disappears after tabbing from "skip to main content link"

### ISSUE THREAD

**Issue by colinpmacarthur** [18F/web-design-standards#483]
**Steps to recreate:**
1. Go to http://pages-staging.18f.gov/usfwds.
2. Hit the tab key until you reach the "Skip to main content" link.
3. Hit the tab key again. The focus disappears.
(If you hit the table key again, the focus will reappear at the download button.)

**Ideal behavior**
After focus arrives at the "Skip to main content" link, the next tab should bring you to the site title. 

**Possible diagnosis:**
No focus styling for links on the library page itself? Perhaps will be resolved by pending PR?

**Found on:**
- Chrome 44, Mac OS 10.10.5
- IE 10, Windows 7 on SauceLabs

**Comment by carodew** [comment]
@angel if you haven't already, can you design a focus state for the page title, which is also a link to the home page?

**Comment by carodew** [comment]
@angel

**Comment by angel** [comment]
Let's use the focus halo we use for everything else and do: 
![image](https://cloud.githubusercontent.com/assets/103496/9856614/c4982468-5ae3-11e5-91fe-7d9413fc1dea.png)

Would that work @carodew?

**Comment by carodew** [comment]
That works for me. I think Colin wrote the original steps to recreate before the homepage was added, so I just want to note that the halo also needs to be added on the standards pages too: https://pages-staging.18f.gov/web-design-standards/getting-started/

**Comment by maya** [comment]
@colinpmacarthur @carodew was this bug for the library site or our homepage? I think this was written before we added in the home page.

**Comment by carodew** [comment]
@maya I think we posted at about the same time. ;)

We'll need it for both places, regardless. Focus should not disappear when tabbing. https://pages.18f.gov/accessibility/checklist/

**Comment by maya** [comment]
@carodew cool just wanted to point out that the homepage image that @angel posted has a focus state around text. It's not a link.

**Comment by carodew** [comment]
Doh! Right then, it should not have a focus state.

On Mon, Sep 14, 2015 at 5:17 PM, Maya Benari <notifications@github.com>
wrote:

> @carodew <https://github.com/carodew> cool just wanted to point out that
> the homepage image that @angel <https://github.com/angel> posted has a
> focus state around text. It's not a link.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/web-design-standards/issues/483#issuecomment-140208389>
> .
>



-- 
Carolyn Dew
carolyn.dew@gsa.gov
470 558 4323
@18F
